cli/server.ts

```diff
@@ -17,7 +17,7 @@ let root = ""
 let dirs = [""]
 let simdirs = [""]
 let docfilesdirs = [""]
-let fileDir = process.cwd()
+let userProjectsDir = path.join(os.homedir(), U.lf("My Coding Projects"));
```

Comment:
Does this folder under the "My Documents" folder? I beleive homedir() points to ".pxt".